Documentation/Input/Pointers.md

```diff
@@ -1,39 +1,239 @@
 Pointer Down / Dragged / Up / Clicked | Raised to report pointer press, drag and release. | [`IMixedRealityPointerHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler)
 Touch Started / Updated / Completed | Raised by touch-aware pointers like [`PokePointer`](xref:Microsoft.MixedReality.Toolkit.Input.PokePointer) to report touch activity. | [`IMixedRealityTouchHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityTouchHandler)
 
-To handle them the corresponding interface must be implemented. Additionally there's the option to receive the events raised on all game objects instead of just the ones on itself and its children. This requires to register as a global event listener calling [`RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem.RegisterHandler*) on the input system with the corresponding interface(s).
-
 > [!NOTE]
 > [`IMixedRealityFocusChangedHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusChangedHandler) and [`IMixedRealityFocusHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusHandler) should be handled in the objects they are raised on. It is possible to receive focus events globally but, unlike other input events, global event handler won't block receiving events based on focus (the event will be received by both global handler and a corresponding object in focus).
 
+#### Example
+
+Below is an example script that changes the color of the attached renderer when a pointer takes or leaves focus or when a pointer selects the object.
+
+```csharp
+  public class ColorTap : MonoBehaviour, IMixedRealityFocusHandler, IMixedRealityPointerHandler
+    {
+        private Color color_IdleState = Color.cyan;
+        private Color color_OnHover = Color.white;
+        private Color color_OnSelect = Color.blue;
+        private Material material;
+
+        private void Awake()
+        {
+            material = GetComponent<Renderer>().material;
+        }
+
+        private void IMixedRealityFocusHandler.OnFocusEnter(FocusEventData eventData)
+        {
+            material.color = color_OnHover;
+        }
+
+        private void IMixedRealityFocusHandler.OnFocusExit(FocusEventData eventData)
+        {
+            material.color = color_IdleState;
+        }
+
+        private void IMixedRealityPointerHandler.OnPointerDown(MixedRealityPointerEventData eventData) {}
+
+        private void IMixedRealityPointerHandler.OnPointerDragged(MixedRealityPointerEventData eventData) { }
+
+        private void IMixedRealityPointerHandler.OnPointerClicked(MixedRealityPointerEventData eventData) 
+        {
+            material.color = color_OnSelect;
+        }
+```
+
+#### Register globally
+
+blah need to add code example for CoreServices.InputSystem.Register etc
```

Comment:
It feels like this section should probably just point to a general "how to register for input events" instead of having to necessarily copy paste and have another define of that... so maybe consider having a reference to somewhere else (which I don't think exists FWIW).

Reply:
Ah I see what you are talking about now